client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -595,15 +576,76 @@ export default function CampaignItemDetails( props: Props ) {
 										) }
 									</div>
 								</div>
-							</div>
-						) }
+							) }
+
+							{ ! is_evergreen && shouldShowStats && (
+								<div className="campaign-item-details__main-stats-row">
+									<div>
+										<span className="campaign-item-details__label">
+											{ is_evergreen && status === 'active'
```

Comment:
@j6ll, I'm in doubt how `is_evergreen` can be `true` (line `585`) inside the condition (line `581`) of `! is_evergreen`.

Reply:
Thanks timur, good spot, I made a tweak and forgot to remove this statement, I will update!

Reply:
Ah, sorry this is a bit messed up.  I actually overwrite all this logic in the next PR #88487.

So if we ignore for now, I plan to merge both branches at the same

Reply:
Thanks for spotting though, completely right :)